### PR TITLE
Toggle lasers

### PR DESCRIPTION
**Pull request by ahoefnagel**
## Added
* LaserManager that toggles laser on button press. 

## Changed
* InteractableManager to a list in the ButtonManager for multi-button assignment support.

### CODE REVIEW

**Review by Jarivanoort — APPROVED** [review]

File: WobblyStealthGame/Assets/Scripts/Interactables/ButtonManager.cs [hunk 1]
```diff
@@ -8,13 +8,13 @@ public class ButtonManager : MonoBehaviour
...
     [SerializeField] private Renderer buttonMaterial;
-    public InteractableManager InteractableManager;
+    public List<InteractableManager> InteractableManager;
 
     private void OnTriggerEnter(Collider other)
     {
         if (other.gameObject.tag == "Player" && other.gameObject.name == "Torso")
         {
-            InteractableManager.ButtonPressed();
+            InteractableManager.ForEach(im => im.ButtonPressed());
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Scripts/Interactables/ButtonManager.cs:17]:
```suggestion
            InteractableManager.ForEach(interactableManager => interactableManager .ButtonPressed());
```

File: WobblyStealthGame/Assets/Scripts/Interactables/StandOnButtonManager.cs [hunk 2]
```diff
@@ -8,14 +8,14 @@ private void OnTriggerEnter(Collider other)
     {
         if (other.gameObject.tag == "Player" && other.gameObject.name == "Torso")
         {
-            InteractableManager.ButtonPressed();
+            InteractableManager.ForEach(im => im.ButtonPressed());
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Scripts/Interactables/StandOnButtonManager.cs:11]:
```suggestion
            InteractableManager.ForEach(interactableManager => interactableManager .ButtonPressed());
```

File: WobblyStealthGame/Assets/Scripts/Interactables/StandOnButtonManager.cs [hunk 3]
```diff
@@ -8,14 +8,14 @@ private void OnTriggerEnter(Collider other)
...
         }
     }
     private void OnTriggerExit(Collider other)
     {
         if (other.gameObject.tag == "Player" && other.gameObject.name == "Torso")
         {
-            InteractableManager.ButtonPressed();
+            InteractableManager.ForEach(im => im.ButtonPressed());
```
Comment (ronanhub) [inline WobblyStealthGame/Assets/Scripts/Interactables/StandOnButtonManager.cs:18]:
```suggestion
            InteractableManager.ForEach(interactableManager => interactableManager .ButtonPressed());
```

**Review by ronanhub — CHANGES_REQUESTED** [review]
code is literally unreadable!!!

**Review by ronanhub — APPROVED** [review]

**Pull request merged**